css/navbar.css

```diff
@@ -0,0 +1,71 @@
+.search-bar {
+  width: 30%;
+  display: flex;
+  box-shadow: var(--box-shadow-bottom);
+}
+
+.search-input {
+  width: 100%;
+  border: 1px solid var(--demo-color-1);
+  border-right: none;
+  padding: 0.5rem;
+  height: 4rem;
+  border-radius: 0.5rem 0 0 0.5rem;
+  outline: none;
```

Comment:
Outline is largely an accessibility feature and should be available on tabbable elements to indicate focus during keyboard navigation..It provides visual feedback for links that have "focus" when navigating a web document using the TAB key (or equivalent). This is especially useful for folks who can't use a mouse or have a visual impairment. If you remove the outline you are making your site inaccessible for these people.

Reply:
Thank you Maruthi, I was not aware of this - will fix it right away